### PR TITLE
Year added

### PR DESCRIPTION
**Pull request by kumuthac23**

### CODE REVIEW

File: src/pageDrawers/CommonMediaDrawer.tsx [hunk 1]
```diff
@@ -332,10 +435,40 @@ function CommonMediaDrawer({
...
+                    }}
+                  >
+                    <Typography>Year:</Typography>
+
+                    <TextField
+                      id="year"
+                      variant="outlined"
+                      size="small"
+                      type="number"
+                      value={year || 2023}
```
Comment (kumuthac23) [inline src/pageDrawers/CommonMediaDrawer.tsx:453]: change this as dynamic

File: src/pageDrawers/CommonMediaDrawer.tsx [hunk 2]
```diff
@@ -353,40 +486,215 @@ function CommonMediaDrawer({
...
+                  {isEdit ? (
+                    <>
+                      {existingMediaImages &&
+                        existingMediaImages.length > 0 &&
+                        existingMediaImages.map((occurance, index) => (
+                          <Box key={`year-${index}`}>
+                            <Typography variant="body1" fontWeight="bold">
+                              Year {occurance.year}
+                            </Typography>
+                            <Divider />
```
Comment (kumuthac23) [inline src/pageDrawers/CommonMediaDrawer.tsx:501]: divider padding

File: src/pageDrawers/CommonMediaDrawer.tsx (hunk 1 above)
Comment (kumuthac23) [reply src/pageDrawers/CommonMediaDrawer.tsx:453]: changed

File: src/pageDrawers/CommonMediaDrawer.tsx (hunk 2 above)
Comment (kumuthac23) [reply src/pageDrawers/CommonMediaDrawer.tsx:501]: fixed

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**